app/validation/validator.py

```diff
@@ -817,8 +928,11 @@ def _validate_schema_contains_submission_page(self, schema, last_block):
         return []
 
     @staticmethod
-    def _error_message(message):
-        return {'message': 'Schema Integrity Error. {}'.format(message)}
+    def _error_message(message, ref=None):
+        error = {'message': f'Schema Integrity Error. {message}'}
+        if ref is not None and isinstance(ref, str):
```

Comment:
The first could be removed.
```suggestion
        if isinstance(ref, str):
```

Reply:
okay